plugin.py

```diff
@@ -89,6 +86,14 @@ def run(cls):
         instance._run()
 
     def __getattr__(self, name):
-        def call(*args, **kwarg):
-            self._call(name, *args)
-        return call
+        if name in ["action", "admin", "cap", "ctcp", "ctcp_reply", "globops", "info", "invite", "ison",
+                    "join", "kick", "links", "list", "lusers", "mode", "motd", "names", "nick", "notice",
+                    "oper", "part", "pass_", "ping", "pong", "privmsg", "quit", "squit", "stats", "time",
+                    "topic", "trace", "user", "userhost", "users", "version", "wallops", "who", "whois",
+                    "whowas", "started", 'privnotice', 'updated']:
```

Comment:
The functions `started` and `updated` are not needed here, since they are only called from bot to plugin. What is this `privnotice` function, I can't find that in the `irc` package.